src/chartmuseum/values.yaml

```diff
@@ -69,7 +69,7 @@ env:
     # disable use of index-cache.yaml
     DISABLE_STATEFILES: false
     # disable Prometheus metrics
-    DISABLE_METRICS: true
```

Comment:
💭 I wonder if we should leave this here for a release so the depreciation warning is printed? Then remove it in a subsequent release?

Reply:
yea.. I am wondering if we should merge this now or wait until v0.15.0 is released. I suppose people could use this chart with `canary` / `main`, so perhaps we keep the old env var here for the time being

Reply:
I want to believe this should have a milestone attached, Lmk if you want me to refactor this back to what it was.

Reply:
@nerdeveloper @jdolitsky I feel like we should wait for v0.15.0 to be released, otherwise it might be a bit weird for both the `ENABLE_METRICS` and the `DISABLE_METRICS` config options to exist together.